src/features/canvassAssignments/components/CreatePlaceCard.tsx

```diff
@@ -38,32 +30,48 @@ export const CreatePlaceCard: FC<AddPlaceDialogProps> = ({
 
   return (
     <Box className={classes.card}>
-      <Card sx={{ width: '90%' }}>
-        <CardContent>
+      <Card sx={{ p: 1, width: '90%' }}>
+        <form
```

Comment:
Great question!

Browsers know how to "submit" a form, which means that it can treat it differently than it treats standalone fields. The browser can then provide functionality like press-enter-to-submit, which is a nice little feature.

When we build javascript web apps, we often don't care because we can re-implement the same functionality (for example in the case of press-enter-to-submit, using an `onKeyUp` listener). But on mobile devices, one of those special things that the browser does when you focus a form field, is that it adds a "submit" button to the touch screen keyboard.

So by adding a form here, and submitting the data in the `onSubmit` handler, a user on mobile can submit the data by tapping a button on their phone keyboard, instead of closing the keyboard and pressing our red button.